packages/material/scss/scheduler/_layout.scss

```diff
@@ -145,8 +145,29 @@
 
         .k-edit-form-container { width: 390px; }
 
-        .k-edit-label { width: auto; }
-        .k-edit-field { width: auto; }
+        .k-edit-label {
+            width: auto;
+            margin-bottom: 0.5rem;
+            label {
+                margin-bottom: 0;
+            }
+        }
+
+        .k-edit-field {
+            width: auto;
+            align-items: center;
+            .k-numerictextbox {
```

Comment:
Since, we change only the input's width that is into a recurrence element container, you can specify the class selector of the `.k-numerictextbox` more precisely. This way, it is safe that if we add another `.k-edit-fields` with `.k-numerictextbox` elements, they will not have leaked styles (70px width).